qml/js/yleApi.js

```diff
@@ -119,7 +119,7 @@ var api = (function(appId, appKey) {
      }
 
      function getProgramsByCategoryId(categoryId, limit, offset) {
-         var url = apiUrl + programsUrl + '&category=' + categoryId + '&availability=ondemand&mediaobject=video&offset=' + offset + '&limit=' + limit;
+         var url = apiUrl + programsUrl + '&category=' + categoryId + '&contentprotection=22-0,22-1&availability=ondemand&mediaobject=video&offset=' + offset + '&limit=' + limit;

```

Comment:
what is `contentprotection`?

Reply:
http://developer.yle.fi/tutorial-not-available-content/index.html says only those with 22-0 or 22-1 are available via the API.